### PR TITLE
Ka/use full strain

### PR DESCRIPTION
**Pull request by kimauth**
Change `LinearElastic` and `Plastic` to use the full strain instead of the strain increment.

### CODE REVIEW

File: src/LinearElastic.jl [hunk 1]
```diff
@@ -31,30 +31,27 @@ function elastic_tangent_3D(E::T, ν::T) where T
...
 """
-    material_response(m::LinearElastic, Δε::SymmetricTensor{2,3}, state::LinearElasticState{3})
+    material_response(m::LinearElastic, Δε::SymmetricTensor{2,3})
 
-Return the stress tensor, stress tangent and the new `MaterialState` for the given strain step Δε such that
+Return the stress tensor and the stress tangent for the given strain ε such that
 
 ```math
 \\boldsymbol{\\sigma} = \\mathbf{E}^\\text{e} : \\Delta \\boldsymbol{\\varepsilon} .
 ```
```
Comment (KnutAM) [inline src/LinearElastic.jl:51]: Remove Δ in function signature and equation?

**Review by KnutAM — COMMENTED** [review]
Remove Δ in function signature in documentation also for Plastic?

File: src/LinearElastic.jl (hunk 1 above)
Comment (kimauth) [reply src/LinearElastic.jl:51]: Good catch, thanks!

**Comment by kimauth** [comment]
Still needs to be changed for `CrystalViscoPlastic`, but I'll merge these so that we can build up on them already.

**Pull request merged**